packages/cloudvision-connector/test/Connector.spec.ts

```diff
@@ -67,27 +76,27 @@ const multiplePathQuery = [
     ],
   },
 ];
-const SEARCH_OPTIONS = {
+const SEARCH_OPTIONS: SearchOptions = {
   search: '',
   searchType: SEARCH_TYPE_ANY,
 };
-const ACTIVE_STATUS = {
+const ACTIVE_STATUS: CloudVisionStatus = {
   code: ACTIVE_CODE,
-  messages: 'Active',
+  message: 'Active',
 };
 const ERROR_MESSAGE = 'error';
-const ERROR_STATUS = {
+const ERROR_STATUS: CloudVisionStatus = {
   code: 3,
-  messages: ERROR_MESSAGE,
+  message: ERROR_MESSAGE,
 };
 
 jest.spyOn(console, 'groupCollapsed').mockImplementation();
 
 describe('getCommandToken', () => {
   test('should return the proper token', () => {
     const conn = new Connector();
-    const command = 'get';
-    const params = {
+    const command: WsCommand = 'get';
```

Comment:
Since we are refactoring, can you reverence the `get` constant instead of having it be a plain string?

Reply:
I don't understand what you mean by this comment.
I can update `const command: WsCommand = 'get'` to `const command = GET`.
Is that what you're referring to?